recipes/src/recipes/RecipeComponent.js

```diff
@@ -0,0 +1,94 @@
+import React,  { useState, useEffect, } from "react";
+
+import styles from '../styles';
+import RecipeList from './RecipeList';
+import RecipeDetail from './RecipeDetail';
+import RecipeForm from './RecipeForm';
+
+import recipesApi from "./recipesApi";
+
+
+const {
+  Container,
+  AddButton,
+} = styles;
+
+const {
+  getRecipes: getRecipesFromDb,
+  deleteRecipe: deleteRecipeFromDb,
+  addOrUpdateRecipe: addOrUpdateRecipeInDb,
+} = recipesApi;
+
+
+function RecipeComponent() {
+
+  const [recipes, setRecipes] = useState([]);
+  const [selected, setSelected] = useState(false);
+  const [updating, setUpdating] = useState(false);
```

Comment:
Maybe `ShowRecipeForm` is a clearer name for this?

Reply:
Possible - I was trying to separate the state from the implementation, but arguably clarity is more important.
I'll change it.